src/main/java/Duke/AddToDoCommand.java

```diff
@@ -0,0 +1,34 @@
+package Duke;
+
+/**
+ * This class encapsulates the command to add a To-Do type task.
+ * It is triggered by the parser and uses the TaskManager, Ui, and Storage.
+ */
+public class AddToDoCommand implements ICommand {
+
+    private final String INPUT;
```

Comment:
Should this be in lowercase since it is not really a constant but a class field?